rest_framework/validators.py

```diff
@@ -42,10 +42,11 @@ class UniqueValidator(object):
     """
     message = _('This field must be unique.')
 
-    def __init__(self, queryset, message=None):
+    def __init__(self, queryset, message=None, ignore_case=True):
```

Comment:
I'd probably prefer to see us do this by adding a `lookup="exact"` parameter here that can be modified.
Make it more explicit what's going on under the hood.